cmd/cloud/deployment_workerqueue.go

```diff
@@ -50,6 +52,23 @@ func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newDeploymentWorkerQueueDeleteCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "delete",
+		Aliases: []string{"de"},
+		Short:   "Delete a Deployment's worker queue",
+		Long:    "Delete a worker queue from an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deploymentWorkerQueueDelete(cmd, args, out)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "The deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "", "", "Name of the deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&name, "name", "n", "", "The name of the worker queue. Queue names must not exceed 63 characters and contain only lowercase alphanumeric characters or '-' and start with an alphabetical character.")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&name, "name", "n", "", "The name of the worker queue.")
```
Don't think we need this information for delete

Reply:
agreed thanks.